text/0201-scope-attributes.md

```diff
@@ -0,0 +1,209 @@
+`Tracer` instances are returned from this functions when the same 
+(name,version,schema_url,attributes) parameters are used.
+```
+
+Since we are defining more precisely previously undefined behavior this is a
+backwards compatible change.
+
+#### Meter
+
+`Get a Meter` API will be extended to add the following parameter:
+
+```
+- `attributes` (optional): Specifies the instrumentation scope attributes to associate
+  with emitted telemetry.
+```
+
+We will modify the following clause:
+
+```
+It is unspecified whether or under which conditions the same or different
+`Meter` instances are returned from this functions.
+```
+
+and replace it by:
+
+```
+The implementation MUST NOT return the same `Meter` when called repeatedly with
+different values of parameters. The only exception to this rule is no-op `Meter`, the
+implementation MAY return the same instance regardless of parameter values.
+
+It is unspecified whether or under which conditions the same or different
+`Meter` instances are returned from this functions when the same 
+(name,version,schema_url,attributes) parameters are used.
+```
+
+#### LogEmitter
+
+`Get LogEmitter` SDK call will be altered to the following:
+
+```
+Accepts the instrumentation scope name and optional version and attributes and
+returns a LogEmitter associated with the instrumentation scope.
+
+The implementation MUST NOT return the same `LogEmitter` when called repeatedly with
+different values of parameters. The only exception to this rule is no-op `LogEmitter`, the
+implementation MAY return the same instance regardless of parameter values.
+
+It is unspecified whether or under which conditions the same or different
+`LogEmitter` instances are returned from this functions when the same
+(name,version,attributes) parameters are used.
+```
+
+### OTLP Changes
+
+The InstrumentationScope message in OTLP will be modified to add 2 new fields:
+attributes and dropped_attributes_count:
+
+```protobuf
+message InstrumentationScope {
+  string name = 1;
+  string version = 2;
+  repeated KeyValue attributes = 3;
+  uint32 dropped_attributes_count = 4;
+}
+```
+
+This change is backwards compatible from OTLP's interoperability perspective. Recipients
+of old OTLP versions will not see the Scope attributes and will ignore them, which we
+consider acceptable from interoperability perspective. This is aligned with our general
+stance on what happens when telemetry sources _add_ new data which old recipients
+don't understand: we expect the new data to be safely ignored.
+
+## Attribute Value Precedence
+
+If the same attribute is specified both at the Span/Metric/LogRecord and at the Scope
+then the attribute value at Span/Metric/LogRecord takes precedence.
+
+This rule applies to non-OTLP exporters in SDKs, to conversions from OTLP to non-OTLP
+formats in the Collector and to OTLP recipients of data that need to interpret the
+attributes in the received data.
+
+## Exporting to non-OTLP
+
```

Comment:
This should call out what to do with conflicts in attributes between instrumentation scope + the "raw" telemetry.

Reply:
I think it is covered by `Attribute Value Precedence` section. Are you looking for more details?

Reply:
@jsuereth Are you looking for more details?